### ISSUE TITLE
the user that reboot is required when system packages is update

### ISSUE THREAD

**Issue by timlau** [timlau/yumex-dnf#39]

**Comment by rapgro** [comment]
:+1:

**Comment by rapgro** [comment]
Maybe try with: dnf needs-restarting

**Comment by timlau** [comment]
@rapgro yes, i need to implement something like that, dnfdaemon uses dnf's extension API, can't use plugins there, but it is not very complex, if is just lookiing for the need restart info in the applied updates and tell the user.

**Comment by rapgro** [comment]
dnf does that with an (optional) plugin and help from tracer. But I do not know how this information could be accessed via the API. You should use tracer directly and reimplement some logic from the plugin, maybe…
